src/auxkernels/Current.C

```diff
@@ -25,6 +25,7 @@ CurrentTempl<is_ad>::validParams()
 
   params.addRequiredCoupledVar("density_log", "The electron density");
   params.addRequiredCoupledVar("potential", "The potential");
+  params.addParam<int>("component", 0, "The component of position. (0 = x, 1 = y, 2 = z)");
```

Comment:
The phrasing of "component of position" is just a bit vague in my opinion

```suggestion
  params.addParam<int>("component", 0, "The component of the Current vector. (0 = x, 1 = y, 2 = z)");
```